README.md

```diff
@@ -6,10 +6,80 @@ If you would like to deploy your own version of covince and need assistance plea
 
 Demo of the code in this repository (with old static data): http://covince.vercel.app
 
-
-# Development
-
-## Install dependencies
+## Configuration
+
+### Top-level sections
+
+* `charts [array]` list of [chart](#chart) definitions
+* `colors [object]` lineages mapped to color definitions
+* `datetime_format [string]` for e.g. data updated date [reference](https://date-fns.org/docs/format) 
+* `map [object]`
+  *  [`settings [object]`](#map-settings)
+  *  [`viewport [object]`](#map-viewport)
+* [`overview [object]`](#overview)
+* `parameters [array]` list of [parameter](#parameter) definitions
+* [`timeline [object]`](#timeline)
+   
+### Chart
+
+* `allow_stack [boolean]` (optional) enables stacking
+* `default_type ['line'|'area']`
+* `heading [string]` 
+* `parameter [string]` should match a parameter id
+* `preset ['percentage']` (optional) applies formatting rules
+* `y_axis [object]` (optional) 
+
+### Chart Y-Axis
+
+* `domain [number[]]` (optional) [reference](https://recharts.org/en-US/api/YAxis#domain)
+* `ticks [string[]]` (optional) [reference](https://recharts.org/en-US/api/YAxis#ticks) 
+* `reference_line [number]` (optional) display a dotted line at the specified value
+* `allow_data_overflow [boolean]` (optional) [reference](https://recharts.org/en-US/api/YAxis#allowDataOverflow)
+
+### Map settings
+
+* `default_lineage [string]` should match a lineage in lists
+* `default_color_by [string]` should match a color in config
```

Comment:
This doesn't need to match a color but a parameter

Reply:
great spot, thanks 👍🏽